### PR TITLE
feat(AMG-14) deploy Talend API Microgateway chart

### PR DESCRIPTION
**Pull request by elogeart**
Version of the Talend API Microgateway chart: 0.1.1

### CODE REVIEW

File: index.yaml [hunk 1]
```diff
@@ -1,9 +1,29 @@
...
+      url: https://www.talend.com/products/
+    name: talend-api-microgateway
+    urls:
+    - https://talend.github.io/helm-charts-public/talend-api-microgateway-0.1.1.tgz
+    version: 0.1.1
   vault-sidecar-injector:
   - apiVersion: v1
     appVersion: 5.0.0
-    created: 2019-12-06T15:48:08.992133996+01:00
+    created: "2019-12-09T08:40:49.2972432+01:00"
```
Comment (sgandon) [inline index.yaml]:
```suggestion
    created: created: 2019-12-06T15:48:08.992133996+01:00
```

File: index.yaml [hunk 2]
```diff
@@ -24,7 +44,7 @@ entries:
     version: 3.0.0
   - apiVersion: v1
     appVersion: 4.1.0
-    created: 2019-12-06T15:48:08.990907576+01:00
+    created: "2019-12-09T08:40:49.2951917+01:00"
```
Comment (sgandon) [inline index.yaml]:
```suggestion
    created: 2019-12-06T15:48:08.990907576+01:00
```

File: index.yaml [hunk 3]
```diff
@@ -1,9 +1,29 @@
...
+    - API
+    - microgateway
+    maintainers:
+    - name: Talend
+      url: https://www.talend.com/products/
+    name: talend-api-microgateway
+    urls:
+    - https://talend.github.io/helm-charts-public/talend-api-microgateway-0.1.1.tgz
+    version: 0.1.1
   vault-sidecar-injector:
```
Comment (pierCo) [inline index.yaml]: @elogeart Do we really use the `vault-sidecar-injector`?
To my understanding: no; Thus we can remove that reference, WDYT?

Comment (asaintsever) [reply index.yaml]: @pierCo No you can't. If you do so then Vault Sidecar Injector chart will not be refrenced anymore then not visible on Helm Hub.

Comment (elogeart) [reply index.yaml]: No we can't remove it, this quite new repo is the whole Talend Helm Charts public registry, not only the one for Talend API Microgateway 😉

Comment (sgandon) [reply index.yaml]: No do not remove it please, this is an open source project that we maintain.

**Comment by elogeart** [comment]
I updated the `index.yaml`: no more 'created' dates changes and no double quotes now.

**Pull request merged**